Dockerfile.template

```diff
@@ -7,11 +7,31 @@ COPY ./build /usr/src/build
 RUN chmod a+x /usr/src/build/install_chromium
 RUN /usr/src/build/install_chromium "%%BALENA_MACHINE_NAME%%"
 
+# Install node
+WORKDIR /usr/local
+
+# renovate: datasource=node-version depName=node
+ARG NODE_VERSION=22.21.1
```

Comment:
I've just remembered that we don't enable renovate in the experimental org, but this is still good to have in place in case it moves to an org where supported.